src/builders/skopeo_builder.rs

```diff
@@ -310,60 +312,52 @@ impl SkopeoSyslinuxBuilder {
         // This temp dir will be removed on dropping.
         let target_dir = TempDir::new("").context("Failed to create temporary directory")?;
 
-        // Copy the container image to a directory
+        // Create a container to obtain full filesystem, instead of dealing with layers.
         Self::run_command(
             &[
-                "skopeo",
-                "copy",
+                "podman",
+                "create",
+                "--replace",
+                "--name",
+                TEMP_CONTAINER_NAME,
                 container_source,
```

Comment:
Does this work with all source formats that also skopeo supports? Otherwise we need to document the changes in behavior

Reply:
Good question!

**TL;DR** Yes, all formats are supported.

According to [containers/image docs](https://github.com/containers/image/blob/620f400df1132d1d693a790baae603fcb175e6e2/docs/containers-transports.5.md?plain=1#L11):

> Tools which use the containers/image library, including skopeo(1), buildah(1), podman(1), **all share a common syntax** for referring to container images in various locations.

I double checked that with `man podman create` and the following `transport:path` formats are supported:

- `docker://docker-reference` (**default**)
- `docker-daemon:docker-reference`
- `dir:path`
- `oci-archive:path:tag`
- `docker-archive:path[:docker-reference]`

It almost fully matches with the documented formats in `gvltctl build`:

https://github.com/gevulotnetwork/gvltctl/blob/aad390de567aa6f8a2fbeae34937ad8d8bb5cc30/src/commands/build.rs#L15-L19

The only difference is `docker-daemon` vs `containers-storage`, but they are both local storages. I additionally tested `podman create` with `containers-storage` - it works :wink:.